src/build/config/config.yaml

```diff
@@ -69,6 +69,9 @@ command:
   check:
     command-role:
       remote: {}
+    lock-remote-required: true
+    lock-required: true
+    lock-type: check
     log-file: false
```

Comment:
Well, what would be the reason to run multiple check commands at the same time? That's not checking anything except that the archiver is working. So it seems important that even with archive-retention to 0, we try to archive to that repo :thinking: 

Maybe if we don't want the check command to fail because another one is already running, we might wait for `archive-timeout` and try again once? Even though I'm not sure it would really worth it (except if users complain about it of course)

Reply:
Another option would be to only make the lock if repo-retention-archive=0. However, I'm OK with leaving this as is for now and see if people complain.